x-pack/plugins/security/server/saved_objects/saved_objects_security_extension.ts

```diff
@@ -218,6 +389,48 @@ export class SavedObjectsSecurityExtension implements ISavedObjectsSecurityExten
       throw this.errors.decorateGeneralError(error, error.body && error.body.reason);
     }
   }
+
+  async authorizeCreate(params: {
+    action: SecurityAction.CREATE | SecurityAction.BULK_CREATE;
+    namespaceString: string;
+    objects: Array<{
+      type: string;
+      id: string;
+      initialNamespaces: string[] | undefined;
+      existingNamespaces: string[] | undefined;
+    }>;
```

Comment:
__nit:__ I think it'd be nice to have a dedicated interface for that object shape (with JSDocs explaining `initialNamespaces` and `existingNamespaces`). Also, wondering if we should rename "namespaces" to "spaces" since Core is already aware of Spaces?

Reply:
I had already created a 'AuthorizeCreateObject' interface, but had failed to update this file with it. I agree with changing 'namespaces' to 'spaces'.

Reply:
> I may be inclined to leave it in places where we expect fields to be populated from the objects' raw docs.

Ah, right, sounds good to me